dts/bindings/firmware/nxp,scmi-cpu.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: System Control and Management Interface (SCMI) cpu domain protocol
```

Comment:
maybe I'm a bit paranoid but can you please add some comment in the description stating that this is a vendor-specific protocol? I know we already have the "nxp" prefix but I want to make sure there's no confusion.